test/test_roscpp/test/CMakeLists.txt

```diff
@@ -1,3 +1,5 @@
+SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -pthread")
```

Comment:
Why is this necessary now? (Same below)

Does that affect downstream packages? Do they need to apply the same change?